runtime/oti/jvminit.h

```diff
@@ -374,6 +374,11 @@ enum INIT_STAGE {
 #define VMOPT_XXENABLEPOSITIVEHASHCODE "-XX:+PositiveIdentityHash"
 #define VMOPT_XXDISABLEPOSITIVEHASHCODE "-XX:-PositiveIdentityHash"
 
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+#define VMOPT_XXENABLEVALHALLA "-XX:+EnableValhalla"
+#define VMOPT_XXDISABLEVALHALLA "-XX:-EnableValhalla"
```

Comment:
`EnableValhalla` -> `EnableValueTypes`.  Name the feature for the particular part thats being enabled / disabled rather than the overall project.  Valhalla might be used for LW10 now, and LW20 in the future.

Reply:
@DanHeidinga I choose the naming to be consistent with the LW1 prototype on the RI. 

Does it matter that the -XX option names are the same?

Reply:
That's a reasonable approach and is better than having conflicting options for testing LW10 functionality.  

The reason to make the runtime flag and option consistent is that it's easier to find the all the references in a single search.